projects/span_suggesters/configs/chunk_suggester.cfg

```diff
@@ -0,0 +1,150 @@
+[paths]
+train = null
+dev = null
+vectors = null
+init_tok2vec = null
+
+[system]
+gpu_allocator = "pytorch"
+seed = 0
+
+[nlp]
+lang = "en"
+pipeline = ["tok2vec","tagger","parser","spancat"]
+batch_size = 128
+disabled = []
+before_creation = null
+after_creation = null
+after_pipeline_creation = null
+tokenizer = {"@tokenizers":"spacy.Tokenizer.v1"}
+
+[components]
+
+[components.parser]
+source = "en_core_web_sm"
+component = "parser"
+
+[components.tagger]
+source = "en_core_web_sm"
+component = "tagger"
+
+[components.tok2vec]
+source = "en_core_web_sm"
+component = "tok2vec"
+
+[components.spancat]
+factory = "spancat"
+max_positive = null
+spans_key = "sc"
+threshold = 0.5
+
+[components.spancat.model]
+@architectures = "spacy.SpanCategorizer.v1"
+
+[components.spancat.model.reducer]
+@layers = "spacy.mean_max_reducer.v1"
+hidden_size = 128
+
+[components.spancat.model.scorer]
+@layers = "spacy.LinearLogistic.v1"
+nO = null
+nI = null
+
+[components.spancat.model.tok2vec]
+@architectures = "spacy.Tok2Vec.v1"
+
+[components.spancat.model.tok2vec.embed]
+@architectures = "spacy.MultiHashEmbed.v1"
+width = 96
+rows = [5000,2000,1000,1000]
+attrs = ["ORTH","PREFIX","SUFFIX","SHAPE"]
+include_static_vectors = false
+
+[components.spancat.model.tok2vec.encode]
+@architectures = "spacy.MaxoutWindowEncoder.v1"
+width = 96
+window_size = 1
+maxout_pieces = 3
+depth = 4
+
+[components.spancat.suggester]
+@misc = "spacy-experimental.ngram_chunk_suggester.v1"
+sizes = [1]
+
+[corpora]
+
+[corpora.dev]
+@readers = "spacy.Corpus.v1"
+path = ${paths.dev}
+max_length = 0
+gold_preproc = false
+limit = 0
+augmenter = null
+
+[corpora.train]
+@readers = "spacy.Corpus.v1"
+path = ${paths.train}
+max_length = 2000
+gold_preproc = false
+limit = 0
+augmenter = null
+
+[training]
+dev_corpus = "corpora.dev"
+train_corpus = "corpora.train"
+seed = ${system.seed}
+gpu_allocator = ${system.gpu_allocator}
+dropout = 0.1
+accumulate_gradient = 1
+patience = 1600
+max_epochs = 0
+max_steps = 20000
+eval_frequency = 200
+frozen_components = ["tok2vec","tagger","parser"]
+annotating_components = ["tagger","parser"]
```

Comment:
This is more complicated and depends on the source pipeline. In this case you need: tagger, parser, attribute_ruler because the noun chunk requires POS and deps.